docs/releases.md

```diff
@@ -8,6 +8,31 @@ title: Releases
 This file contains changes and migration steps for the Graphback project. 
 Please follow individual releases for more information.
 
+# 0.12.0
+
+##### New Features
```

Comment:
Not really the format we were using but it is ok to change it

Reply:
Essentially the same but using a title - easier to read.